FWCore/Framework/interface/EDConsumerBase.h

```diff
@@ -117,10 +121,16 @@ namespace edm {
                                          std::map<std::string, ModuleDescription const*> const& labelsToDesc,
                                          std::string const& processName) const;
 
+    void esModulesWhoseProductsAreConsumed(
+        eventsetup::EventSetupProvider const&,
+        std::array<std::vector<eventsetup::ComponentDescription const*>*,
+                   static_cast<unsigned int>(Transition::NumberOfEventSetupTransitions)>& esModules) const;
```

Comment:
Given how often `static_cast<unsigned int>(Transition::NumberOfEventSetupTransitions)` appears in the code, maybe a `constexpr` should be defined maybe in the file holding Transition?

```C++
constexpr unsigned int kNumberOfEventSetupTransitions = static_cast<unsigned int>(Transition::NumberOfEventSetupTransitions);
```

Reply:
Done. Better. Thanks.